torch/_torch_docs.py

```diff
@@ -13722,6 +13722,58 @@ def merge_dicts(*dicts):
 """,
 )
 
+add_docstr(
+    torch.Generator.graphsafe_set_state,
+    r"""
```

Comment:
probably just need to add these new methods here https://github.com/pytorch/pytorch/blob/ea8f6e2e54be2101fe2fe0c243d93fac9995aca9/torch/_C/__init__.pyi.in#L1389

Reply:
Fixed